sycl/source/detail/global_handler.cpp

```diff
@@ -189,43 +201,54 @@ void GlobalHandler::unloadPlugins() {
     }
   }
   // Clear after unload to avoid uses after unload.
-  GlobalHandler::instance().getPlugins().clear();
-}
-
-void GlobalHandler::drainThreadPool() {
-  if (MHostTaskThreadPool.Inst)
-    MHostTaskThreadPool.Inst->drain();
+  getPlugins().clear();
 }
 
 void shutdown() {
+  GlobalHandler *handler = nullptr;
```

Comment:
```suggestion
  GlobalHandler *Handler = nullptr;
```

Reply:
fixed in https://github.com/intel/llvm/pull/7908/commits/9e296153178ccb5d9fa6ba4c2793cc34be97f979